src/lib/init.ts

```diff
@@ -62,6 +63,24 @@ export const init = (optionsBase?: InitProps) => {
     return;
   }
 
+  /**
+   * This establishes a connection to the prismatic url and preloads
+   * assets (css, js, fonts, etc.) into browser cache. Subsequent
+   * calls, use existing connections and cached assets.
+   */
+  if (options.preload) {
```

Comment:
Is there a reason we're making this configurable? Seems like it should just be the behavior of `init`

Reply:
Did it for possible performance issue from the consumer, although this isn't render blocking it does't it does defer the dispatch of the main pages on-load event until after its own on-load event is dispatched.

Reply:
Alright. If you think making this configurable is potentially helpful I'm game for it. However, optional properties with `true` defaults aren't ideal. Can you change this to `skipPreload?: boolean`?

Reply:
Good call, doing now.